WcaOnRails/db/migrate/20180822165331_add_wca_marketing_team.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class AddWcaMarketingTeam < ActiveRecord::Migration[5.2]
+  def change
+    execute <<-SQL
+      INSERT INTO teams (friendly_id, email, rank, created_at, updated_at) values ('wmt', 'marketing@worldcubeassociation.org', 20, NOW(), NOW());
```

Comment:
Why is this ranked 20? We use alphabetical order here, so the WCA Marketing Team would be between 50 and 60.

Reply:
I had put it right below the WCT to gain visibility.